docs/lesson/lesson.yaml

```diff
@@ -147,7 +147,15 @@ paths:
                 title: Title of the lesson
                 description: Some information about this lesson
                 author: 64a33e71eea95284f397a6ee
-                attachments: [ 63ed1cd25e9d781cdb6a6b15 ]
+                attachments:
+                  - _id: 63ec1cd51e9d781cdb6f4b14
+                    author: 648afee884936e09a37deaaa
+                    fileName: EnglishB2Level
+                    link: 'https://english-b2-test.com'
+                    description: This is basic B2 level English test.
```

Comment:
As I understood we don't have a description now. So delete it.

Reply:
fixed